<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="usersComboBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>231</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="refreshButton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>40</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Frissítés</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuKezel_s">
    <property name="title">
     <string>Kezelés</string>
    </property>
    <addaction name="manageProfiles"/>
   </widget>
   <addaction name="menuKezel_s"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="manageProfiles">
   <property name="text">
    <string>Fiókok kezelése</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
